starters/lit-mobx-vaadin/src/components/counter-example/starter-counter.ts

```diff
@@ -0,0 +1,21 @@
+import { html, css } from 'lit';
```

Comment:
By looking at other starters, I think it would be beneficial to split pages/routes, from components. It would be great to have somthing like /pages/CounterPage.js and a components/Counter.js